app/src/main/java/com/courseproject/tindar/entities/Message.java

```diff
@@ -0,0 +1,63 @@
+package com.courseproject.tindar.entities;
+
+import java.sql.Timestamp;
+
+public class Message {
+    private String messageID;
+    private String text;
+    private Timestamp timestamp;
+    private String sentFrom;
+    private String sentTo;
+    private String conversationId;
```

Comment:
okay it bothers me that it's Id everywhere here, and ID everywhere in the other file.

Reply:
Also: let's not make conversationID an attribute; conversation handles what messages are in it so we don't need to do it both ways